test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -58,7 +58,7 @@ public void PackCommand_IncludeExcludePackageFromNuspec()
   </metadata>
   <files>
     <file src=""contentFiles/any/any/image.jpg"" target=""\Content\image.jpg"" />
-    <file src=""/contentFiles/any/any/image2.jpg"" target=""Content\other\image2.jpg"" />
```

Comment:
It ends up pointing to the root of the drive, so it won't find the file.  I added that test when I made it point to the project root but that wasn't right.